src/states/game/game.gd

```diff
@@ -143,6 +150,32 @@ func wait_for_timer(time: float) -> void:
 	await get_tree().create_timer(time, false).timeout
 
 
+func play_sound(sound: SoundEffect, is_left: bool = true) -> void:
```

Comment:
What's there is totally fine, but you can also say `is_left := true` which is the same thing as `is_left: bool = true`